web/pdf_page_view.js

```diff
@@ -490,10 +490,11 @@ var PDFPageView = (function PDFPageViewClosure() {
     beforePrint: function PDFPageView_beforePrint() {
       var pdfPage = this.pdfPage;
 
-      var viewport = pdfPage.getViewport(1);
+      var viewport = pdfPage.getViewport(2);
       // Use the same hack we use for high dpi displays for printing to get
       // better output until bug 811002 is fixed in FF.
-      var PRINT_OUTPUT_SCALE = 2;
+      // In this fork I have not tested for regressions here
```

Comment:
Remove this line as it is not useful for the code itself. Before we land this, we must have tested for regressions anyway.